release/servicing/alpine/docker/Dockerfile

```diff
@@ -25,7 +25,7 @@ ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell/$PS_INSTALL_VERSION
 RUN mkdir -p ${PS_INSTALL_FOLDER}
 
 # Unzip the Linux tar.gz
-RUN tar zxf /tmp/linux.tar.gz -C ${PS_INSTALL_FOLDER}
+RUN tar zxf /tmp/linux.tar.gz -C ${PS_INSTALL_FOLDER} -v
```

Comment:
can you validate the `-v` works at the end?

Reply:
It has been working
https://github.com/PowerShell/PowerShell-Docker/blob/549c3ab8e78cc58ba0a1f84470e996766569ae32/release/stable/alpine/docker/Dockerfile#L28